semigroups.cabal

```diff
@@ -128,6 +136,9 @@ library
     if flag(bytestring)
       build-depends: bytestring >= 0.9 && < 1
 
+    if flag(bytestring-builder) && impl(ghc < 7.8)
```

Comment:
Is the `impl(ghc < 7.8)` condition really accurate? I.e. are there really no install-plans for GHC >= 7.8 which can lack the respective instances?